VideoPlayer.js

```diff
@@ -213,6 +214,11 @@ export default class VideoPlayer extends Component {
     // This triggers channel Avatar Channel & Follow Button [Landscape View]
     this.props.streamLandscapeStore.isShadowOverlayOn = true;
 
+    // Seeking to previous location
+    if(this.props.streamStore.seekVideo){
+      this.seekTo(this.props.streamStore.seekToTime)
+    }
```

Comment:
I know your prettier doesn't work on this but can we make a habit of doing this for spaces:
```js
if () {